### ISSUE TITLE
Improve color contrast in "teal white" scheme

### ISSUE THREAD

**Issue by james-nash** [buildit/gravity-particles#30]
**Describe the bug**
The contrast between `grav-co-grp-b-neutral-subtle` and `grav-co-grp-a-neutral` is less than 4.5:1 and this causes a11y tests on our pattern library to fail (specifically, the `<cite>` element - see buildit/gravity-ui-web#369).

You can see the exact contrast ratios in our [pattern library's colour scheme tables](http://style-next.buildit.digital/components/detail/color-schemes.html):
![image](https://user-images.githubusercontent.com/16718916/70424530-724faf80-1a67-11ea-901a-07ab840d861b.png)

**Additional context**
The short term fix would be to pick another shade of grey for `grav-co-grp-b-neutral-subtle`, which achieves the 4.5:1 contrast ratio (AA) or greater.

However, longer term we need to:
* Review our contrast requirements. So far, we only require 3:1 on all colour combos, which is sufficient to meet AA _for large text_. This was because achieving 4.5:1 on _all_ combos was proving difficult to achieve. However, perhaps we need to revisit that and define a subset of combos that need to be 4.5:1.
* In any case, we should add some automated tests to this repo, so that updates to colour scheme can be checked quickly, without needing to get them into the pattern library first.
